hsmodels/schemas/aggregations.py

```diff
@@ -373,20 +377,21 @@ class Config:
 
     abstract: str = Field(default=None, title="Abstract", description="A string containing a summary of a aggregation")
 
-    _parse_abstract = root_validator(pre=True, allow_reuse=True)(parse_abstract)
+    _parse_abstract = model_validator(mode='before')(parse_abstract)
 
 
 class TimeSeriesMetadata(TimeSeriesMetadataIn):
     type: AggregationType = Field(
-        const=True,
+        frozen=True,
         default=AggregationType.TimeSeriesAggregation,
         title="Aggregation type",
         description="A string expressing the aggregation type from the list of HydroShare aggregation types",
-        allow_mutation=False,
```

Comment:
I wrote the PR for allow_mutation, sad to see my contribution to pydantic dissolved.  I guess frozen will do.